buildSrc/build.gradle.kts

```diff
@@ -10,6 +10,14 @@ plugins {
     id("com.github.autostyle")
 }
 
+gradlePlugin {
+    plugins {
+        create("releaseNotePlugin") {
+            id = "org.postgres.releasenotes"
+            implementationClass = "org.postgresql.buildtools.ReleaseNotesTask"
+        }
+    }
+}
```

Comment:
`gradlePlugin` is not really needed here.
So far all classes from `buildSrc/src/main/java` are visible in all `build.gradle.kts` files outside of `buildSrc`

Reply:
Thanks I have to confess to not having a clue how this works